js_modules/dagit/packages/core/src/assets/AssetEntryRoot.tsx

```diff
@@ -50,59 +48,27 @@ export const AssetEntryRoot: React.FC<RouteComponentProps> = ({location, match})
     return <Redirect to={pathname} />;
   }
 
-  const pathDetails = () => {
-    if (currentPath.length === 1 || view !== 'directory') {
-      return <Link to="/instance/assets">Assets</Link>;
-    }
-
-    const breadcrumbs: BreadcrumbProps[] = [];
-    currentPath.slice(0, currentPath.length - 1).reduce((accum: string, elem: string) => {
-      const href = `${accum}/${encodeURIComponent(elem)}`;
-      breadcrumbs.push({text: elem, href});
-      return href;
-    }, '/instance/assets');
-
-    return (
-      <Box flex={{direction: 'row', alignItems: 'center'}} style={{maxWidth: 500}}>
-        <Box margin={{right: 4}}>
-          <Link to="/instance/assets">Assets</Link>:
-        </Box>
-        <Breadcrumbs
-          breadcrumbRenderer={({text, href}) => (
-            <Link to={href || '#'}>
-              <span style={{fontSize: '14px'}}>{text}</span>
-            </Link>
-          )}
-          items={breadcrumbs}
-        />
-      </Box>
-    );
-  };
-
   return (
     <Page>
-      <Group direction="column" spacing={20} padding={{horizontal: 24}}>
-        <PageHeader
-          title={
-            view !== 'directory' ? (
-              <Heading>{currentPath[currentPath.length - 1]}</Heading>
-            ) : (
-              <Box flex={{alignItems: 'center'}}>
-                {currentPath
-                  .map<React.ReactNode>((p, i) => <Heading key={i}>{p}</Heading>)
-                  .reduce((prev, curr, i) => [
-                    prev,
-                    <Box key={`separator_${i}`} padding={4}>
-                      <IconWIP name="chevron_right" size={24} />
-                    </Box>,
-                    curr,
-                  ])}
-              </Box>
-            )
-          }
-          icon="asset"
-          description={<PathDetails>{pathDetails()}</PathDetails>}
-        />
+      <PageHeader
+        title={
+          view !== 'directory' ? (
+            <Heading>{currentPath[currentPath.length - 1]}</Heading>
+          ) : (
+            <Box flex={{alignItems: 'center', gap: 4}}>
```

Comment:
I think in directory mode not being able to click through the path details (which had links to each folder I think) will be a bit of a regression... could we make the Headings rendered on line 60 links?

Reply:
What if we add this to the metadata beneath the header, e.g. above `Latest materialization`? I think it's already not a great experience to have the full path listed out in the header, especially if it's super long.

Reply:
Nm, I'll put it back. I don't want to get sucked into spending too much time on this now. :P